### PR TITLE
Added shipping ups method templates #7063

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

**Comment by gautampanday** [comment]
nereid webshopshipping-ups

**Comment by tarunbhardwaj** [comment]
please follow color theme

File: templates/webshop/checkout/delivery_method.jinja [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                        </span>
+                      </label>            
+                    </p>
+                    {% endfor %}
+                  </div>
+                </div>
+                <button type="submit" class="btn btn-warning btn-lg btn-block" id="payment-btn">
+                  Continue to Payment <i class="fa fa-angle-double-right"></i>
+                </button>
+              {% elif sale.require_custom_shipping_quote %}
```
Comment (tarunbhardwaj) [inline templates/webshop/checkout/delivery_method.jinja]: no such field `require_custom_shipping_quote` ignore this block

**Comment by tarunbhardwaj** [comment]
LGTM

**Pull request merged**